iwyu_output.cc

```diff
@@ -1123,17 +1123,22 @@ void ProcessFullUse(OneUse* use,
   // (B3) Discard symbol uses for builtin symbols, including new/delete.
   // TODO(csilvers): we could use getBuiltinID(), but it returns
   // non-zero for things like malloc.  Figure out how to use it.
-  if (const FunctionDecl* fn_decl = DynCastFrom(use->decl())) {
+  if (GetLocation(use->decl()).isInvalid()) {
+    VERRS(6) << "Ignoring use of " << use->symbol_name() << " ("
+             << use->PrintableUseLoc() << "): built-in symbol\n";
+    use->set_ignore_use();
+    return;
+  } else if (const FunctionDecl* fn_decl = DynCastFrom(use->decl())) {
     if (StartsWith(use->symbol_name(), "__builtin_")) {
-      VERRS(6) << "Ignoring use of " << use->symbol_name()
-               << " (" << use->PrintableUseLoc() << "): built-in function\n";
+      VERRS(6) << "Ignoring use of " << use->symbol_name() << " ("
```

Comment:
It would make the patch easier to parse if you didn't (let clang-format) reformat these unchanged lines.

Reply:
Yep. Sorry for that